<launch>
    <rosparam command="load" file="$(find jackal_bsc)/launch/bsc_params.yaml" ns="jackal_bsc"/>
    <node name="twist_stamped_publisher" package="jackal_bsc" type="twist_stamped_publisher"/>
    <node name="bsc_node" package="jackal_bsc" type="bsc_node"/>
</launch>